doc/tasks/phpcsfixer2.md

```diff
@@ -97,12 +97,13 @@ The tool will fix all files if the tool version has changed or the list of fixer
 Cache is supported only for tool downloaded as phar file or installed via composer.
 
 
-**path_mode**
+**can_intersect**
 
-*Default: null*
-
-Specify path mode (can be override or intersection).
+*Default: true*
 
+Intersection mode can only be used when you have a configuration file which contains a Finder.
+This mode works best since only files that are being commit and are in your configuration will be checked.
+When there is no Finder in your configuration, all committed files will be checked.
```

Comment:
when there is no finder in configuration, setting path-mode=intersection will crash the execution